### ISSUE TITLE
0.8.8.7 does not render text

### ISSUE THREAD

**Issue by iamkhush** [summernote/django-summernote#289]
Summernote Version `0.8.8.7` 
Django 2.0.6

Form - 
`body = forms.CharField(label=_('message'), widget=SummernoteInplaceWidget())`

Template - as you can see we are using crispy form.
```
<form method="post" class="form-horizontal" enctype="multipart/form-data">
  {% crispy form %}
</form>

<script type="text/javascript">
  document.addEventListener("DOMContentLoaded", function (event) {
    initSummernote();
  });
</script>
```
Settings - 
```
SUMMERNOTE_CONFIG = {
        'lazy': True,
        'iframe': False
}
```

Issue - 
When page loads. I can see the content in the body textarea field.
When `initSummernote` executes, the `textarea` with name `body` gets `display:none` , the toolbar loads and html is not rendered.

Screenshot After loading -
![screenshot from 2018-06-13 16-39-31](https://user-images.githubusercontent.com/404460/41358380-90600632-6f28-11e8-8994-ec1c2d660cb7.png)

The same configuration is working with 0.8.8.6

**Comment by iamkhush** [comment]
I think this is might be fixed with https://github.com/summernote/django-summernote/pull/287

**Comment by lqez** [comment]
Closed by #287, thanks.
